scripts/apply-fixes.js

```diff
@@ -0,0 +1,25 @@
+const glob = require("glob");
+const fs = require("fs");
+
+glob.sync("themes/default/content/registry/packages/**/_index.md").forEach(path => {
+
+    // https://github.com/pulumi/registry/issues/5095
+    // Finds and replaces an errant set of backticks in the TF docs.
+    if (path === "themes/default/content/registry/packages/rancher2/api-docs/nodetemplate/_index.md") {
+        const content = fs.readFileSync(path, "utf8");
+        const lines = content.split("\n");
+        let precedingLine, nextLine = undefined;
+
+        lines.forEach((line, i) => {
+            if (line.endsWith("implements `fixNodeTemplateID()` that will update tfstate with proper id.")) {
+                precedingLine = i;
+                nextLine = i + 1;
+            }
+        });
+
+        if (precedingLine && nextLine && lines[nextLine] === "") {
+            delete lines[nextLine];
```

Comment:
```suggestion
if (lines[nextLine].includes("```") {
    delete lines[nextLine];
}
```

maybe something like this would suffice for now if i'm following the logic correctly here. At least this would stop us from accidentally deleting the wrong thing when/if the upstream changes.